bin/workspace.code-workspace

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Usually you add such files on the root folder, not in a subfolder. I would move this file one level up.